std/stdio.d

```diff
@@ -5134,7 +5134,7 @@ if (is(typeof(copy(data, stdout.lockingBinaryWriter))))
 }
 
 /*********************
- * Thrown if I/O errors happen.
+ * Thrown if I/O exceptions happen.
```

Comment:
"errors" here has its usual meaning, not a D throwable.

Reply:
reverted.